recommender/tasks/neural_networks.py

```diff
@@ -22,6 +26,26 @@ def execute_rl_agent_training():
     rl_agent_training()
 
 
+@celery.task
+def execute_training(_=None):
+    if Config.AGENT_VERSION == PRE_AGENT:
```

Comment:
please use this syntax:
`from flask import current_app; current_app.config[ENV_VARIABLE]`
It's cleaner and autmatically chooses the config based on env (testing/prod/dev/global)